dtschema/schemas/i2c/i2c-controller.yaml

```diff
@@ -44,6 +44,17 @@ properties:
       Number of nanoseconds the SCL signal takes to rise; t(r) in the I2C
       specification.
 
+  i2c-scl-clk-low-timeout-ms:
+    description:
+      Number of miliseconds the clock line needs to be pulled down in order
+      to force a waiting state.
+
+  i2c-scl-has-clk-low-timeout:
+    description:
+      Boolean value that indicates whether the controller implements the
+      feature of wait induction through SCL low, with the timeout being
+      implemented internally by the controller.
+
   i2c-sda-falling-time-ns:
     description:
```

Comment:
at the bottom allOf required excluding them from being used together:
https://lore.kernel.org/all/20230118163208.GA117919-robh@kernel.org/

```
allOf:
  - not:
      required:
        - i2c-scl-has-clk-low-timeout
        - i2c-scl-clk-low-timeout-ms
```

Reply:
Thanks!